src/query/functions/temporal/aggregation_test.go

```diff
@@ -353,3 +353,10 @@ func TestUnknownAggregation(t *testing.T) {
 	_, err := NewAggOp([]interface{}{5 * time.Minute}, "unknown_agg_func")
 	require.Error(t, err)
 }
+
+func TestHoltWinters(t *testing.T) {
+	holtWintersFn := makeHoltWintersFn(0.2, 0.6)
+	val := holtWintersFn([]float64{1, math.NaN(), 5, 10, 15})
```

Comment:
Could you do something like `holtWintersFn([]float64{math.NaN(),  1, math.NaN(), 5, 10, 15, math.NaN(), math.NaN()})` as your test data to hit all the NaN specific logic at once?